lib/landing_session.js

```diff
@@ -118,7 +120,7 @@ class LandingSession extends Session {
   getRebaseSuggestion(subjects) {
     const { upstream, branch } = this;
     let command = `git rebase ${upstream}/${branch} -i`;
-    command += ' -x "git node land --amend"';
+    command += ' -x "git-node land --amend"';
```

Comment:
Keep `git node`

Reply:
Sure, but since the command is now different why do we keep suggesting the old one? Or am I missing something?

Reply:
The command shouldn't be different, git will turn any executable named `git-<command>` into a git command (`git <command>`). Did you get any errors trying to run `git node` (when installing with npm, because when developing `git node` won't work because the executable won't be in the path)?